### ISSUE TITLE
[Feature Request] Reset all Metadata for a book/series

### ISSUE THREAD

**Issue by MKH-42** [gotson/komga#485]
### Is your feature request related to a problem? Please describe.
It should be possible to reset all Metadata for a book or series with one click.

### Describe the solution you'd like
In best case you can choose in the menue an item to reset all metadata for a book or series.
After conformation all data should be reseted.

**Comment by gotson** [comment]
Can you describe clearly what you mean by reset metadata?

**Comment by MKH-42** [comment]
Every book or series could have metadata like titel, author, release date, summery, status, read state, publisher, genre, tags, also the locked status.
It would be nice to reset or clear this data with one click.
So all fields are empty and could be refilled (manually or by an scan)

**Comment by gotson** [comment]
Not sure what purpose it would serve?

If you want to edit manually, just edit manually.

If you want to refresh the metadata, there is already a function for that.

**Comment by MKH-42** [comment]
Hello gotson,

yes it is possible to fill it manually.
And yes, it is possible to do it manually.

The idea behind this is to make it easier.

2 examples:
I already added metadata to an series.
Now I found out, the information like tags, summery, authors, release date was not optimal.
So I start again to fill all meta data with the better description.
At the moment I move the series on an other place, than make a rescan of the directory to clean all metadata.
Than I move the directory back to the right place. 
Than I start again to fill in all metadata.
This is possible but nicer should be a button to reset all data for books or series.

Other example: I filled in automaticly via API metadata. I found out, I made a mistake in the script.
Now I can go forward like in example 1 (move, scan, move back) to make sure, to delete ALL metadata.
Or we have a small menu item.

I think this feature has not the highest priority but I made a lot of experiments with metadata in the last days and found out, this feature would help :)

Maybe there is an posibility via API to reset all metadata and not only to set the data to "".
Only a small wish, nothing urgent :-)

MKH

**Comment by gotson** [comment]
I don't understand. In both your examples you are going to refresh metadata after resetting, so there's no point of resetting. Or did I miss something?

**Comment by MKH-42** [comment]
The help here is to reset ALL metadata because there was a mistake to fill out.
To make sure, to clean up all metadata it would be faster to have one button/menu item to reset all metadata with one click. 

metadata has some entries like date, tags, year, volume, authors ... and I want to make sure, I don't forget to delete an item.
And sometime I don't wont to fill the data again till I have a better quality of the metadata - so I want to let the data empty.

**Comment by jonaballen** [comment]
+1
I would like to be able to refresh metadata on a library/series level, and if metadata options have been changed the information should get updated according to new settings or have an option to be reset before refreshing.

Scenario that I have that I think I share with others:
I had a large library of comics before installing komga and setting it up. The books were from different sources and collected over time, many books either lack meta-data or had fan-made metadata or scraped metadata. Since I liked the notion of using metadata I used that as my default.
However the result was inconsistent naming of Book titles, sometimes filenames, in the other meta-data and some case meta-data that didnt fit my preferences. However I didnt notice this directly sinceI have over 14k books and 4k series, but over time I noticed the inconsistencies in naming. 
So I figured it would be better for me to use filename as title, since I have always kept that in order on disk.
Then I consider refreshing metadata to replace comicinfo.xml titles with filename. But apparently this doesnt work since 

> metadata import is additive and not subtractive

I want to keep my reading history and all manual edits, so rather not remove and rescan the libraries.
